diff --git a/node_modules/gift-pegjs/index.d.ts b/node_modules/gift-pegjs/index.d.ts
index 1bc90ff..a30a8d3 100644
--- a/node_modules/gift-pegjs/index.d.ts
+++ b/node_modules/gift-pegjs/index.d.ts
@@ -110,6 +110,7 @@ interface Question {
   title: string | null;
   stem: TextFormat;
   hasEmbeddedAnswers: boolean;
+  choices: TextChoice[] | NumericalChoice[];
   globalFeedback: TextFormat | null;
 }
 
@@ -119,6 +120,7 @@ export interface Description {
   type: Extract<QuestionType, "Description">;
   title: string | null;
   stem: TextFormat;
+  choices: TextChoice[] | NumericalChoice[];
   hasEmbeddedAnswers: boolean;
 }
 
@@ -126,6 +128,7 @@ export interface Category {
   id?: string | null;
   tags?: string[] | null;
   type: Extract<QuestionType, "Category">;
+  choices: TextChoice[] | NumericalChoice[];
   title: string;
 }
 
